### PR TITLE
Allow to configure the `QueryDefaultsPlugin`

### PR DESCRIPTION
**Pull request by ruudk**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | https://github.com/php-http/client-common/pull/67
| Documentation   | https://github.com/php-http/documentation/pull/243
| License         | MIT

#### What's in this PR?

Allow to configure the `QueryDefaultsPlugin`

#### Checklist

- [x] Updated CHANGELOG.md to describe BC breaks / deprecations | new feature | bugfix
- [x] Documentation pull request created (if not simply a bugfix)

### CODE REVIEW

**Comment by ruudk** [comment]
Anything I need to change to get this merged? 😊

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -390,6 +390,19 @@ private function createClientPluginNode()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('query_defaults')
+                    ->canBeEnabled()
+                    ->info('Set query parameter to default value if it does not exist.')
```
Comment (xabbuh) [inline DependencyInjection/Configuration.php]: > Sets query parameters to default value if they are not present in the request.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
 
+## 1.13.0 - 2018-11-08
```
Comment (xabbuh) [inline CHANGELOG.md]: should be `## Unreleased` instead

File: DependencyInjection/Configuration.php [hunk 3]
```diff
@@ -390,6 +390,19 @@ private function createClientPluginNode()
...
+                    ->fixXmlConfig('parameter')
+                    ->children()
+                        ->arrayNode('parameters')
+                            ->info('List of query parameters. Names and values must not be url encoded as this plugin will encode them.')
```
Comment (xabbuh) [inline DependencyInjection/Configuration.php]: [...] as the plugin [...]

**Comment by dbu** [comment]
thanks a lot!

**Comment by dbu** [comment]
tagged 1.13.0 with this

**Pull request merged**